core/src/main/java/com/devonfw/tools/solicitor/ruleengine/drools/ModelHelper.java

```diff
@@ -200,8 +200,8 @@ public static boolean match(String input, String condition) {
     }
     if (input != null && condition != null) {
       if (condition.startsWith(REGEX_PREFIX)) {
-        deprecationChecker.check(true,
-            "Use of 'REGEX:' prefix notation is deprecated, use '(REGEX)' suffix instead. See https://github.com/devonfw/solicitor/issues/78");
+        deprecationChecker.check(false,
+            "Use of 'REGEX:' prefix notation is deprecated, use '(REGEX)' suffix instead. See https://github.com/devonfw/solicitor/issues/263");
```

Comment:
We should list both tickets as the first gives some background and the second indicates that it is now stage 2.